### PR TITLE
Add abbreviations to the labels

### LINKED ISSUE #81: Extend labels with an emoji and optionally also a short version
In the eat-api Website and the Apps, there were defined lists for the ingredients with custom emojis, to display instead of the longer names.
I would propose to add these to the `labels.json` as well, to save the doubled amount of work, oposed to handle these lists in every application by itself.

Further, I would also propose to add a short version, abbreviation or something for labels, where there exists no emoji. As it would be easier to display these, instead of the enum Labels. Example: `BAVARIA` can be `GQB`.

### PR DESCRIPTION
**Pull request by Philipp000**
Resolves #81 

Here I added the emojis for the dishes as previously used on the website to the enum for the labels.
I only added one field for a short form, as I think it would be too much, having an emoji as well as a short form.
Further, if there are any suggestions/additons/changes on the used emojis I'm happy to adapt them.

I also thought about setting the abbreviation to the name, when there is nothing given, but in my opinion this is not necessary, and therefore in the output it only shows either none or the specified abbreviation.

### CODE REVIEW

File: src/entities.py [hunk 1]
```diff
@@ -226,63 +226,64 @@ def to_api_representation(self) -> Dict[str, object]:
...
+    MUSTARD = {Language.DE: "Senf"}, None
+    SESAME = {Language.DE: "Sesam"}, None
+    SULPHURS = {Language.DE: "Schwefeldioxid"}, "🔶"
+    SULFITES = {Language.DE: "Sulfite"}, "🔻"
+    LUPIN = {Language.DE: "Lupine"}, None
+    MOLLUSCS = {Language.DE: "Weichtiere"}, "🐙"
+    SHELL_FRUITS = {Language.DE: "Schalenfrüchte"}, None
+
+    BAVARIA = {Language.DE: "Zertifizierte Qualität Bayern"}, "GQB"
+    MSC = {Language.DE: "Marine Stewardship Council"}, "MSC"
```
Comment (COM8) [inline src/entities.py:264]: How about 🎣 instead of MSC?
```suggestion
    MSC = {Language.DE: "Marine Stewardship Council"}, "🎣"
```

Comment (Philipp000) [reply src/entities.py:264]: Sounds good

**Comment by Philipp000** [comment]
What do you think about those labels, where currently no symbol is set?
We could also introduce short abbreviations, like the first letter of the name or something.

**Comment by COM8** [comment]
Yes, this would be useful. For this I'm currently using the "old" Studentenwerk abbreviations.

https://github.com/TUM-Dev/Campus-UWP/blob/94b1700aefba382d61cc747043853f2a7cbee50f/ExternalData/Classes/Manager/DishManager.cs#L41-L97

**Comment by Philipp000** [comment]
This sounds good, I have updated the abbreviations accordingly

**Review by COM8 — APPROVED** [review]
Thanks.

**Pull request merged**